flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -489,6 +489,29 @@
 	 */
 	public static final String MESOS_RESOURCEMANAGER_TASKS_CPUS = "mesos.resourcemanager.tasks.cpus";
 
+	/**
+	 * The base container image to use for task managers (started via the unified containerizer).
```

Comment:
Please rephrase the comment: "The container image to use for task managers."    In other words, it isn't a 'base' image in this context, and numerous containerizers are now supported.